docs/tutorials/misc/300_otcp.ipynb

```diff
@@ -0,0 +1,780 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    ax.scatter(
</pre><p>There's a lot of code repeat for plotting (here, above when just visualizing the data and also below for the forward approach).</p><p>Think it would be great to define a common plotting function which can optionally accept the conformal candidates.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>